log10/_httpx_utils.py

```diff
@@ -29,6 +30,24 @@
 httpx_async_client = httpx.AsyncClient(timeout=timeout)
 
 
+class LLM_PROVIDER(Enum):
```

Comment:
nit: LLM_PROVIDER could be misleading here. maybe `LLM_CLIENTS` or `LLM_PYTHON_CLIENTS`? 
e.g. code example `client = openai.OpenAI()`.